lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -353,7 +361,7 @@ The following message sequence chart shows the bootstrap process:
     Application<<="LwM2M carrier Library"      [label="LWM2M_CARRIER_EVENT_CONNECTING"];
     Application<<="LwM2M carrier Library"      [label="LWM2M_CARRIER_EVENT_CONNECTED"];
     Application<<="LwM2M carrier Library"      [label="LWM2M_CARRIER_EVENT_BOOTSTRAPPED"];
-    Application<<="LwM2M carrier Library"      [label="LWM2M_CARRIER_EVENT_READY"];
```

Comment:
`LWM2M_CARRIER_EVENT_LTE_READY` will be given between BOOTSTRAPPED and REGISTERED.